test/servico/cst23Test.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package servico;
+
+import auxiliar.Perfil;
+import datamapper.PopulateDB;
+import datamapper.exceptions.NonexistentEntityException;
+import java.text.ParseException;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import modelo.UsuarioModel;
+
+/**
+ *
+ * @author prgoes
+ */
+public class cst23Test {
+    
+    
+    @BeforeClass
+    public static void setUpClass(){ 
+        //
+    }   
+    
+    @AfterClass
+    public static void tearDownClass() {
+        //
+    }
+    
+    @Before
+    public void setUp() throws Exception {
+        PopulateDB.recreateDB("prosub", "root", "");
+        PopulateDB.populateUseCaseTest();
+        
+    }
+    
+    @After
+    public void tearDown() {
+        //
+    }
+
+    @Test
+    public void testeEditarUsuario() throws ParseException, Exception {
+
+        LoginService loginService = new LoginService();
+        assertTrue(loginService.verificarUsuarioESenha(
+                "Administrador", "123456"));
+        AdministracaoDeUsuariosService administracaoDeUsuarioService
+                = new AdministracaoDeUsuariosService();
+        List<UsuarioModel> usuariosList
+                = administracaoDeUsuarioService.listarUsuarios();
+        UsuarioModel usuario = usuariosList.get(3);

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['3' is a magic number.](https://www.codacy.com/app/calebepb/substituicao/file/2777729608/issues/source?bid=3364462&fileBranchId=3364484#l64)